frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -34,6 +84,11 @@ export const ScheduledEventsTableRow: FunctionComponent<ScheduledEventsTableRowP
           ? `Every ${props.interval} ${props.interval_type}`
           : 'None'}
       </td>
+      <td>
+        <Button variant="outline-danger" size="sm" onClick={handleClick} data-id={props.id}>
+          {isLoading ? 'Loading…' : 'Run'}
```

Comment:
`Run Now` ...

Reply:
Seeing where `isLoading` is being used, I don't think that's the right name ... `isRunning` seems more appropriate. Or do we go for `Creating instance` :)

Reply:
They are both good options, I think `Creating instance` is better :)